projects/fgia_accv2022_1st/README.md

```diff
@@ -0,0 +1,76 @@
+# Solution of FGIA ACCV 2022(1st Place)
+
+This is the 1st Place Solution for Webly-supervised Fine-grained Recognition, refer to the ACCV workshop competition in https://www.cvmart.net/race/10412/base.
```

Comment:
Better indicate this is the fine-tuning solution. For pre-training, you can put a link pointing to MMSelfSup.

Reply:
done